azure-tests/src/main/java/fixtures/azurereport/AutoRestReportServiceForAzure.java

```diff
@@ -28,7 +28,7 @@
     private final AutoRestReportServiceForAzureService service;
 
     /** server parameter. */
-    private String host;
+    private final String host;
```

Comment:
I mgmt, I've renamed this to `endpoint`.

Reply:
`endpoint` is probably a more accurate name.